test/new_rlecore_tb.py

```diff
@@ -0,0 +1,75 @@
+# !/usr/bin/env python3
+# This is the module for testing the RLEcore.
+
+from litex.gen import *
+
+from litex.soc.interconnect.stream import *
+from litex.soc.interconnect.stream_sim import *
+
+from litejpeg.core.common import *
+from litejpeg.core.rle.rlecore import RunLength
+
+from common import *
+
+# Testbanch for the RLEcore module.
+
+"""
+Under this module a matrix containing 64 blocks is been sent as an
+input to the RLE core and the output is been printed and compared
+with the one from the reference modules.
+In this way the result from the RLEcore is been verified.
+"""
```

Comment:
This module takes a matrix containing 64 blocks of XXXX and verifies the RLECore produces the same output as the reference data.

Reply:
Done.